specification/monitor/resource-manager/microsoft.insights/stable/2018-04-16/scheduledQueryRule_API.json

```diff
@@ -0,0 +1,794 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Microsoft Insights API",
+    "version": "2018-04-16",
+    "x-ms-code-generation-settings": {
```

Comment:
You shouldn't have `x-ms-code-generation-settings` in the actual swagger file (I don't think we even use this anymore at all).

Reply:
Fixed.